common/recurrence_test.go

```diff
@@ -17,34 +17,45 @@ limitations under the License.
 package common
 
 import (
+	"github.com/smartystreets/goconvey/convey"
 	"testing"
 	"time"
-
-	"github.com/stretchr/testify/assert"
 )
 
 func TestParseExclusionDate(t *testing.T) {
-	exDateString := "EXDATE:20060102T150405Z,20180510T021030Z"
-
-	dates, err := ParseExclusionDates([]string{exDateString})
-	if err != nil {
-		t.Fatal(err)
-	}
-	assert.Equal(t, 2, len(dates))
-
-	first := dates[0]
-	assert.Equal(t, 2006, first.Year())
-	assert.Equal(t, time.January, first.Month())
-	assert.Equal(t, 2, first.Day())
-	assert.Equal(t, 15, first.Hour())
-	assert.Equal(t, 4, first.Minute())
-	assert.Equal(t, 5, first.Second())
-
-	second := dates[1]
-	assert.Equal(t, 2018, second.Year())
-	assert.Equal(t, time.May, second.Month())
-	assert.Equal(t, 10, second.Day())
-	assert.Equal(t, 2, second.Hour())
-	assert.Equal(t, 10, second.Minute())
-	assert.Equal(t, 30, second.Second())
+	convey.Convey("Given an exclusion date", t, func() {
```

Comment:
I've never heard of convey testing framework? any reason we're using it?

Reply:
It allows writing behavior driven test cases. Plus it has a nice UI where you can see the your "test stories", which part of your is covered by tests etc, coverage increment/decrement since last run, past test runs statistics etc. http://goconvey.co/

Reply:
that's pretty neat. are there others using this? how does it compare to Ginko?

Reply:
Both are behavior driven testing framework. I tried both Ginko and Convey and felt Convey is more natural to use and has a nice UI. Also it is more popular than Ginko